tests/gcp/hooks/test_google_discovery_api.py

```diff
@@ -20,15 +20,16 @@
 import unittest
 from unittest.mock import call, patch
 
-from airflow import configuration, models
+from airflow import models
+from airflow.configuration import load_test_config
 from airflow.gcp.hooks.discovery_api import GoogleDiscoveryApiHook
 from airflow.utils import db
 
 
 class TestGoogleDiscoveryApiHook(unittest.TestCase):
 
     def setUp(self):
-        configuration.load_test_config()
+        load_test_config()
```

Comment:
I thought I got rid of all of these. This isn't needed any longer and cna just be removed entirely.
```suggestion
```

Reply:
Yeah. There are a few tests still where they are left :)

Reply:
I think that was a comment to earlier commit. It's been fixed in the fixup.